src/legacy/core_plugins/console/np_ready/public/legacy.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import 'brace';
+import 'brace/ext/language_tools';
+import 'brace/ext/searchbox';
+import 'brace/mode/json';
+import 'brace/mode/text';
+
+/* eslint-disable @kbn/eslint/no-restricted-paths */
+import { npSetup, npStart } from 'ui/new_platform';
+import uiRoutes from 'ui/routes';
+import { DOC_LINK_VERSION } from 'ui/documentation_links';
+import { I18nContext } from 'ui/i18n';
+import { ResizeChecker } from 'ui/resize_checker';
+import 'ui/autoload/styles';
+import 'ui/capabilities/route_setup';
+/* eslint-enable @kbn/eslint/no-restricted-paths */
+
+import template from '../../public/quarantined/index.html';
+import { CoreSetup } from '../../../../../core/public';
+
+interface SetupRegisterAppArgs {
+  id: string;
+  mount: (ctx: any, targetElement: HTMLDivElement) => Promise<any>;
```

Comment:
FYI this interface slightly changed. The second parameter is now a object of named values. `element` is the div. This would be closer:

```ts
mount: (ctx: any, { element }: { element: HTMLElement }) => Promise<any>;
```

Reply:
Thanks updated!